pkg/utils/emails/contacts.go

```diff
@@ -0,0 +1,72 @@
+package emails
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/sendgrid/sendgrid-go"
+)
+
+const (
+	sgHost     = "https://api.sendgrid.com"
+	sgContacts = "/v3/marketing/contacts"
+)
```

Comment:
What made you choose to have this here rather than in the `sendgrid` package?

Reply:
I think having them in the `sendgrid` package makes more sense; I was planning on shoring up the contacts in SC-13281 so that both the email builders and the marketing list related code can use them. It might make more sense to move the package up a level to the `utils` as well.